docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -142,6 +183,198 @@ data:
     [“172.16.0.1”]
 ```
 
+## Debugging DNS resolution
+
+### Create a simple Pod to use as a test environment
+
+Create a file named busybox.yaml with the following contents:
```

Comment:
Include this example as an `{% include code.html ... %}` statement as well, please.

Reply:
@chenopis The Travis build: https://travis-ci.org/kubernetes/website/builds/310360212?utm_source=github_status&utm_medium=notification

Reply:
done